includes/wikia/GlobalFunctions.php

```diff
@@ -1557,3 +1557,18 @@ function wfGetBaseDomainForHost( $host ) {
 
 	return $wgWikiaBaseDomain;
 }
+
+function wfDoesURLHaveFandomDomain( $url ): bool {
```

Comment:
short comment about what this method does and that is should work on all env would be nice.
+ maybe renaming it to something like `wfIsFadomUrl`.

Reply:
`wfIsFandomUrl` was my first idea but I think the current name is better because it doesn't need the explanation, it just checks if the url has a fandom domain :man_shrugging: The fact that it works on all envs should be obvious, no? I'd add `TODO` if it didn't :wink: